samples/boards/nrf52/power_mgr/src/main.c

```diff
@@ -71,6 +71,16 @@ void main(void)
 			sys_pm_ctrl_disable_state(SYS_POWER_STATE_CPU_LPS_1);
 			break;
 
+		case 7:
+			printk("\n<-- Enabling %s state --->\n",
+				       STRINGIFY(SYS_POWER_STATE_CPU_LPS_1));
+			sys_pm_ctrl_enable_state(SYS_POWER_STATE_CPU_LPS_1);
+
+			printk("<-- Forcing %s state --->\n",
+				STRINGIFY(SYS_POWER_STATE_CPU_LPS_2));
+			sys_pm_force_power_state(SYS_POWER_STATE_CPU_LPS_2);
```

Comment:
All earlier added lines with `STRINGIFY` were aligned in the way that the power state names started in the same column as in the following `sys_pm_...` function call. Although the beauty of such alignment may be disputable, I think the convention should be kept, for consistency. Same applies to line 102.

Reply:
Fixed. Thank you.